client/src/components/organizations/OrganizationSwitchPopover.vue

```diff
@@ -2,52 +2,101 @@
-    <ion-list class="organization-list">
-      <ion-item
-        class="organization-list__item body"
-        button
-        lines="none"
-        v-for="org in connectedOrgs"
-        :key="org.handle"
-        :disabled="org.active"
-        @click="onOrganizationClick(org)"
+    <div class="current-organization">
+      <div class="current-organization-content">
+        <ion-avatar class="organization-avatar">
+          <span v-if="!isTrialOrg">{{ currentOrg ? currentOrg.id.substring(0, 2) : '' }}</span>
+          <ms-image
+            v-else
+            :image="LogoIconGradient"
+            class="organization-avatar-logo"
+          />
+        </ion-avatar>
+        <ion-text class="organization-name title-h4">
+          {{ currentOrg?.id }}
+        </ion-text>
+      </div>
+      <!-- <div
+        class="current-organization-buttons"
+        v-show="userInfo && userInfo.currentProfile == UserProfile.Admin"
       >
-        <div class="organization">
-          <ion-avatar class="organization-avatar">
-            <span>{{ org.id.substring(0, 2) }}</span>
-          </ion-avatar>
-          <div class="organization-text-content">
-            <ion-label class="organization-text-content__name">
-              <span class="body">
-                {{ org.id }}
-              </span>
-            </ion-label>
-            <ion-label class="organization-text-content__email">
-              <span class="body-sm">
-                {{ org.userLabel }}
-              </span>
-            </ion-label>
+        will be manage later
+        <ion-text
+          class="button-medium custom-button custom-button-fill"
+          button
+          @click="onInviteClick"
+        >
+          <ion-icon :icon="personAdd" />
+          {{ $msTranslate('OrganizationSwitch.invite') }}
+        </ion-text>
+        <ion-text
+          class="button-medium custom-button custom-button-fill"
+          button
+          @click="onManageOrganizationClick"
+        >
+          <ion-icon :icon="cog" />
+          {{ $msTranslate('OrganizationSwitch.manage') }}
+        </ion-text>
+      </div> -->
+    </div>
+    <div
+      class="connected-organization"
+      v-if="filteredOrgs.length > 0"
+    >
+      <ion-title class="connected-organization-title body">{{ $msTranslate('OrganizationSwitch.loggedInOrgs') }}</ion-title>
+      <ion-list class="organization-list">
+        <ion-item
+          class="organization-list__item body"
+          button
+          lines="none"
+          v-for="org in filteredOrgs"
+          :key="org.handle"
+          @click="onOrganizationClick(org)"
+        >
+          <div class="organization">
+            <ion-avatar class="organization-avatar">
+              <span v-if="!isTrialOrg">{{ org.id.substring(0, 2) }}</span>
+              <ms-image
+                v-else
+                :image="LogoIconGradient"
+                class="organization-avatar-logo"
+              />
```

Comment:
From a trial organization with a standard org connected in the popover, the logo is too big and applies to the non-trial org
![Screenshot 2024-09-10 at 13 27 30](https://github.com/user-attachments/assets/14991837-7dcf-40b9-953f-c5d7d0ab6bf7)

From a standard organization with a trial org connected, we see `tr` in place of the parsec logo for the trial
![Screenshot 2024-09-10 at 13 28 37](https://github.com/user-attachments/assets/ea8a1361-ac32-4404-b6e7-f3292a28a96a)


Currently the current org's `trial` status is applied to every organization listed. I don't think you have an easy way to check for every organization listed if it's a trial or not, so maybe another approach would be necessary. You could use either 2 letters or the logo for every organization and open an issue for a function that retrieves the `trial` status of any non-current organization so that we could have such a feature.